packages/components/psammead-image/src/index.jsx

```diff
@@ -32,13 +32,39 @@ const StyledImg = styled.img`
   width: 100%;
 `;
 
+const getMimeType = srcset => {
+  if (!srcset) return null;
+
+  const firstSrcsetUrl = srcset.split(',')[0].split(' ')[0];
```

Comment:
```suggestion
  // ?. captures an edge case where the srcset passed in is null or invalid
  const [ firstSrcSet ] = srcset?.split(',');
  const [ firstSrcSetUrl ] = firstSrcSet?.split(' ');
```
This reads a bit more clearly imo. The main bit is we need to be a bit defensive if the src set is invalid, so null or contains an unexpected format. I believe if the srcset was invalid in this way the page would not render at all with the code as it is; I'd write a test to cover this eventuality.

I'd agree with @jroebu14 that this logic may be better to be in Simorgh, is that where the srcset is constructed; you could pass the mime type in as a prop?

Reply:
I'll move this logic over to Simorgh and pass through the mime-types as props for each of the srcsets